File: src/sdk-reference/kuzzle/constructor/snippets/constructor.js
const options = {
  host: 'localhost',
  autoResubscribe: false
};

// Instantiate the SDK with an embedded protocol
const kuzzleWS = new Kuzzle('websocket', options);

// Instantiate the SDK with a custom protocol
const customProtocol = new MyCustomProtocol();
const kuzzleCustom = new Kuzzle(customProtocol, options);

File: src/sdk-reference/cpp/1/kuzzle/constructor/snippets/constructor.cpp
std::string hostname = "kuzzle";
kuzzleio::WebSocket* ws = new kuzzleio::WebSocket(hostname);

kuzzleio::options options;
options.auto_resubscribe = false;

kuzzleio::Kuzzle *kuzzle = new kuzzleio::Kuzzle(ws, options);

std::cout << "New SDK instance successfully created" << std::endl;
